### ISSUE TITLE
Bug appears

### ISSUE THREAD

**Issue by froggypaule** [DataStewardshipPortal/dsp-common#293]
### What web browser is used when the issue appears?

Firefox

### How are you using DSW while encountering the issue?

- [ ] Testing on localhost (dsw-deployment-example)
- [ ] Using DSW managed by someone else (specify in text)
- [X] Using DSW that I manage

### What parts of DSW are affected?

- [ ] Dashboard
- [ ] Documents
- [ ] Document Template Editors
- [ ] Document Templates
- [ ] Locales
- [X] Knowledge Model Editors
- [ ] Knowledge Models
- [ ] Project Importers
- [ ] Projects
- [ ] Settings
- [ ] Users and Authentication

### Version

```plain text
Client
Version: v4.11.1~83a636e
Built at: 2024-10-08T14:56:29Z
---
Serveur
Version: v4.11.1~cddefe42
Built at: 2024-10-08T17:10:22Z
---
Document Worker
Version: v4.11.0~02f06ba
Built at: 2024-10-01T05:26:04Z
---
Mailer
Version: v4.11.0~02f06ba
Built at: 2024-10-01T05:26:13Z
```

### What happened?

Wrt to ["Fixed an issue with squashing in KM Editor where some changes would disappear"](https://changelog.ds-wizard.org/#v4-11-1-backend) I am afraid we still experience these problems.

This morning we did some km editing, in particular introducing the item select type questions in our km. Most of the time, if not all the time, when we either edited/saved the km settings, or published the km, we lost either question titles, or the selection of the item select question, or the tag settings, or the text. It does not only happen for item select type questions, but it is there that we struggled the most with lost data. 

Thank you!

**Comment by krystofkomanec** [comment]
Hi Paulette, thanks for reporting! We will look into it.

**Comment by krystofkomanec** [comment]
Hi @froggypaule, indeed you discovered another bug with squashing. It should be fixed now in [4.11.2 backend](https://changelog.ds-wizard.org/#4-11-2)

**Comment by froggypaule** [comment]
Hello @krystofkomanec ! I've seen you posted a fix (4.11.2): I'll let you know as we'll have a working session tomorrow. Cheers! (and I just read your msg here...).

**Comment by froggypaule** [comment]
I confirm that all is good now! thanks again :). Cheers!
